datalad/core/local/resulthooks.py

```diff
@@ -0,0 +1,177 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Interface utility functions
+
+"""
+
+__docformat__ = 'restructuredtext'
+
+import logging
+import json
+
+lgr = logging.getLogger('datalad.core.local.hooks')
```

Comment:
Quickly grepping this code base, I wasn't able to spot anywhere else that the logger doesn't follow the module name, so `s/hooks/result&/`?

Reply:
Forgotten leftover, will fix. Thx!

Reply:
Done.